backend/src/utils/config.ts

```diff
@@ -26,6 +28,12 @@ export interface Config {
     jwks: string
   }
 
+  httpClient: {
+    proxy: string
+    noProxy: string[]
+    defaultOpts: AgentOptions
```

Comment:
A lot of unit tests are failing because they can't find this value, you should be able to resolve it by adding it to `backend/src/utils/__mocks__/config.ts`